ixmp/tests/report/test_operator.py

```diff
@@ -236,18 +236,26 @@ def test_store_ts(caplog: pytest.LogCaptureFixture, test_mp: "Platform") -> None
 
     # A message is logged
     r = caplog.record_tuples[-1]
+    error = (
```

Comment:
This is a case where the shim *could* catch and re-raise the exception in order to satisfy the test expectation, instead of adjusting the test. It's still passable to do the latter.

I hope there is little or no downstream code that relies on the specific class and text of this exception; if there is, it can be adjusted following this example.